provider/clusterrosa/hcp/resource.go

```diff
@@ -578,6 +583,21 @@ func (r *ClusterRosaHcpResource) Create(ctx context.Context, request resource.Cr
 		}
 	}
 
+	if common.HasValue(state.WaitForStdComputeNodesComplete) &&
```

Comment:
What happen if user set `state.WaitForCreateComplete` to true? do we need to enforce waiting for cluster ready first?

Reply:
It waits first because it comes first in the section above

Reply:
I could see an issue, let's say the user signals to wait only for the compute nodes and the cluster goes to error. It would timeout but it might no be clear why

I guess we could enforce that for this to be enabled the wait for cluster also needs to be

Reply:
Yes, I agree with this approach, thanks